src/Microsoft.AspNetCore.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -147,6 +149,13 @@ public class LinkTagHelper : UrlResolutionTagHelper
         [HtmlAttributeName(FallbackHrefAttributeName)]
         public string FallbackHref { get; set; }
 
+        /// <summary>
+        /// Boolean value that determines if an integrity hash will be compared with <see cref="FallbackHref"/> value.
+        /// Value defaults to <c>true</c> if not provided.
+        /// </summary>
+        [HtmlAttributeName(FallbackIntegrityCheckAttributeName)]
+        public bool FallbackIntegrityCheck { get; set; } = true;
```

Comment:
Should we invert this instead? `SuppressFallbackIntegrityCheck`?

Reply:
Figured @DamianEdwards and folks decided on the current name after discussing potential inversions. Maybe they didn't? I'm also a fan of the "SupressX" but they wanted this attribute

Reply:
@DamianEdwards we have precedence for using the `SuppressX` for our compat shim items. This would fit right in 👍

Reply:
If we do have precedence, then let's stick with the pattern.

Reply:
Personally I don't prefer double negation. We have these in compat switches and the experience is confusing and not immediately clear.

Reply:
Although I understand the logic for the "SupressX" prefix, intuitively it makes more sense to use the "asp-fallback-" prefix because it relates to the other attributes that contain that prefix.

Reply:
My beef with `asp-fallback-integrity-check` is that most of the time you'll end up doing: `<link ... asp-fallback-integrity-check="false" />` when you want to use the attribute. Where as if you named it `SuppressX` it would enable users to write: `<link ... asp-suppress-fallback-integrity />` (clearer).

Reply:
Valid point. I may be in the minority that explicitly sets attribute values. If the majority of developers  would call the attribute without explicitly setting the value (using the default) then your naming convention makes more sense.